src/sys.zig

```diff
@@ -698,7 +699,8 @@ pub fn mkdiratZ(dir_fd: bun.FileDescriptor, file_path: [*:0]const u8, mode: mode
     return switch (Environment.os) {
         .mac => Maybe(void).errnoSysP(syscall.mkdirat(@intCast(dir_fd.cast()), file_path, mode), .mkdir, file_path) orelse .success,
         .linux => Maybe(void).errnoSysP(linux.mkdirat(@intCast(dir_fd.cast()), file_path, mode), .mkdir, file_path) orelse .success,
-        else => @compileError("mkdir is not implemented on this platform"),
+        .windows => @compileError("mkdir is not implemented on this platform"),
+        .wasm => @compileError("mkdir is not implemented on this platform"),
```

Comment:
Why use duplicate cases instead of `.windows, .wasm =>`?

Reply:
i only like to do `.a, .b` when the `=>` is on the next line generally

Reply:
doesn't justify code duplication imo